web/components/unstake-diko-modal.tsx

```diff
@@ -107,7 +112,12 @@ export const UnstakeDikoModal = ({ showUnstakeModal, setShowUnstakeModal, staked
           minimumFractionDigits: 2,
           maximumFractionDigits: 6,
         })}{' '}
-        stDIKO.
+        stDIKO which equals to{' '}
+        {microToReadable(stakedAmount * stDikoToDiko).toLocaleString(undefined, {
```

Comment:
I think this is fine, not sure if there can be slight rounding errors or an issue when a user unstakes & withdraws and gets less than the UI showed.... probably not?